drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -3895,8 +3895,13 @@ VkRenderPass RenderingDeviceVulkan::_render_pass_create(const Vector<AttachmentF
 		render_pass_create_info.pDependencies = nullptr;
 	}
 
-	render_pass_create_info.correlatedViewMaskCount = 1;
-	render_pass_create_info.pCorrelatedViewMasks = &correlation_mask;
+	if (p_view_count == 1) {
+		render_pass_create_info.correlatedViewMaskCount = 0;
```

Comment:
I would add a comment here that validation layers indicate that if multiview is not used, Corrolated View Masks must not be set.

Reply:
Comments added about both validations.